common/pom.xml

```diff
@@ -34,6 +34,8 @@ under the License.
 
     <modules>
         <module>rya.api</module>
+        <module>rya.api.model</module>
+        <module>rya.api.function</module>
```

Comment:
Based on the contents of this module, it seems like rya.api.evaluation would a better name.  Naming the module function suggests that the module consists entirely of Filter Functions.  Just a nit tho.

Reply:
Done.